app/models/item.rb

```diff
@@ -1,4 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :wishlist
+  has_one :reservation, dependent: :destroy
```

Comment:
Would add an empty line after. Don't really remember whether we separate those in other models.